lib/http/preprocessors.js

```diff
@@ -62,12 +62,12 @@ const sessionHandler = ({ Sessions, Users, Auth, bcrypt }, context) => {
       });
 
   // Standard Bearer token auth:
-  } else if (!isBlank(authHeader) && authHeader.startsWith('Bearer')) {
+  } else if (!isBlank(authHeader) && authHeader.startsWith('Bearer ')) {
```

Comment:
what's going to happen if I separate `Bearer` and value by any other character than space? like `Bearer=token_value`